imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -100,30 +104,29 @@ class CaseExplorer extends Component {
           // Adding the latest update time to the case for easier sorting later
           unitDesc.cases.push(
             Object.assign({
-              latestUpdate: caseUpdateTimeDict[caseIdStr] || 0,
+              latestUpdate: caseUpdateTimeDict[caseIdStr] || (new Date(caseItem.creation_time)).getTime(),
               unreadCounts: caseUnreadDict[caseIdStr]
             }, caseItem)
           )
         }
         return dict
       }, {})
-
-      // Creating a case grouping *array* from the unit dictionary
+      const sortType = sortBy ? sorters[sortBy] : sorters[SORT_BY.LATEST_UPDATE]
       return Object.keys(unitsDict).reduce((all, unitTitle) => {
         const { bzId, cases, unitType } = unitsDict[unitTitle]
-
-        // Sorting cases within a unit by the order descending order of last update
         cases.sort((a, b) => b.latestUpdate - a.latestUpdate)
+        // Sorting cases within a unit by the order descending order of last update
+        const sortedCases = sortBy ? cases.sort(sorters[sortBy]) : cases
```

Comment:
Why are you not just using `sortType`? The default sort for the inner cases lists should be LATEST_UPDATE too